ci/e2e.sh

```diff
@@ -4,7 +4,7 @@
 make setup
 
 # Run the e2e tests, save exit code for later
-DOCKER_REGISTRY=${DOCKER_REGISTRY:=localhost:32000} E2E_FLAGS="--test.v" make -j e2e
+TEST_HELM_REPO_URL=https://raw.githubusercontent.com/bookingcom/shipper/${TRAVIS_COMMIT}/test/e2e/testdata DOCKER_REGISTRY=${DOCKER_REGISTRY:=localhost:32000} E2E_FLAGS="--test.v" make -j e2e
```

Comment:
Suggested change:

```
TEST_HELM_REPO_URL=${TEST_HELM_REPO_URL:https://raw.githubusercontent.com/bookingcom/shipper/${TRAVIS_COMMIT}/test/e2e/testdata}
```

This makes `TEST_HELM_REPO_URL` use the existing environment variable if it exists, but otherwise use `https://raw.githubusercontent.com/bookingcom/shipper/${TRAVIS_COMMIT}/test/e2e/testdata` as default. That's important for us to overwrite it from inside our internal CI pipeline, so we can point it to whatever chart repo we want.